src/content/docs/auto-telemetry-pixie/pixie-data-security-overview.mdx

```diff
@@ -9,18 +9,25 @@ tags:
 metaDescription: 
 ---
 
-New Relic hosts Pixie, our open source observability tool for Kubernetes applications. Pixie fetches data flowing through a systems kernel via [eBPF](https://ebpf.io/) probes from a privileged state. If an application contains sensitive data, this data flow can potentially collect that data in plain text and save it to [NRDB](/docs/telemetry-data-platform/get-started/nrdb-horsepower-under-hood/). For example, Pixie gives insight into any data on your system, including raw HTTP requests in the kernel, allowing you to view all headers, query parameters, and HTTP bodies of a request.
+Auto-telemetry with Pixie is New Relic One's integration of Community Cloud for Pixie, a managed version of Pixie open source software. Auto-telemetry with Pixie therefore benefits from Pixie's approach to keeping data secure. The data that Pixie collects is stored entirely within your Kubernetes cluster. This data is not persisted outside of your environment, and will never be stored by Community Cloud for Pixie. This means that your sensitive data remains within your environment and control.
 
-New Relic, on the other hand, fetches and stores a subset of data that only revolves around an application's performance. These performance metrics include, but are not limited to: 
+Community Cloud for Pixie makes queries directly to your Kubernetes cluster to access the data. In order for the query results to be shown in the Community Cloud for Pixie UI,CLI, and API, the data is sent to the client from your cluster using a reverse proxy. 
+
+Community Cloud for Pixie’s reverse proxy is designed to ensure:
+* Data is ephemeral. It only passes through our cloud proxy in transit. This ensures data locality.
+* Data is encrypted and secure while in transit. Only you are able to read your data. 
```

Comment:
Do you read data while it's in transit? Or only when it's arrived at point A or point B?